src/providers/twitch/ChatroomChannel.cpp

```diff
@@ -0,0 +1,44 @@
+#include "ChatroomChannel.hpp"
+
+#include <QDebug>
+#include "TwitchApi.hpp"
+#include "common/Common.hpp"
+#include "messages\Emote.hpp"
```

Comment:
this backslash in the include looks off, change it to a normal forward slash

Reply:
fixed